### PR TITLE
added translations and editing produced

### PR DESCRIPTION
**Pull request by Neodelf**
Hello, this commit adds some translations and a bit formatted text to enhance readability.
I hope this helps someone.

### CODE REVIEW

File: README-ruRU.md [hunk 1]
```diff
@@ -3683,9 +3682,9 @@
 
   linux_organization = Organization.find(...)
 
-  # плохо - нарушает private области видимости
+  # плохо - нарушает приватные области видимости
   linux_organization.send(:reset_token)
-  # хорошо - выдаст exception
+  # хорошо - выдаст исключение
```
Comment (arbox) [inline README-ruRU.md]: Please change the notes to the parens style, we try to write `bad (some description)` in the whole guide. We still have a lot of inconsistencies, but let's try to keep this style.

File: README-ruRU.md [hunk 2]
```diff
@@ -2523,7 +2522,7 @@
 * <a name="visibility"></a>
   Ограничивайте область видимости методов (`private`, `protected`) в зависимости
   от их планируемого применения. Не оставляйте все в области `public` (это
-  стандартное значение). В конце концов мы пишем на *Руби*, а не на *Питоне*.
+  стандартное значение). В конце концов мы пишем на *Ruby*, а не на *Python*.
   <sup>[[ссылка](#visibility)]</sup>
```
Comment (arbox) [inline README-ruRU.md]: This guide provides the only spelling "Руби". Any discussions about that could be done privately if needed.

Comment (Neodelf) [inline README-ruRU.md]: Do you mean that changes not need?

File: README-ruRU.md (hunk 1 above)
Comment (Neodelf) [inline README-ruRU.md]: Yes, i will do that

File: README-ruRU.md (hunk 2 above)
Comment (arbox) [inline README-ruRU.md]: Yes, we would prefer the cyrillic spelling, please leave this point unchanged.

**Comment by arbox** [comment]
Thank you! You provide real help in the project!

**Pull request merged**